src/audio/Audio.js

```diff
@@ -327,7 +327,7 @@ class Audio extends Object3D {
 
 	onEnded() {
 
-		this.isPlaying = false;
+		this.stop();
```

Comment:
Calling `stop()`  could cause a warning if the audio has no playback control.

To be on the safe side, I would simply add `this._progress = 0;` to fix the issue.

Reply:
I agree. I didn’t consider the playbackControl warning. I’ve updated the fix and pushed the commit again. dfe6c8c